tests/main/snap-discard-ns/task.yaml

```diff
@@ -1,12 +1,14 @@
 summary: check basic operation of snap-discard-ns
+
 details: |
     The internal command snap-discard-ns discards (unmounts) the
     /run/snapd/ns/$SNAP_NAME.mnt file and removes the current mount profile
     /run/snapd/ns/snap.$SNAP_NAME.fstab. The profile removal is optional and it
     is not an error if it doesn't exist.
+
 prepare: |
-    . $TESTSLIB/snaps.sh
-    install_local test-snapd-tools
+    snap install test-snapd-tools
```

Comment:
Why is this no longer doing a local install?

Reply:
@mvo5, you are right, thanks for that catch